libcloud/compute/drivers/cloudstack.py

```diff
@@ -654,6 +654,12 @@ def list_nodes(self, project=None):
             args['projectid'] = project.id
         vms = self._sync_request('listVirtualMachines', params=args)
         addrs = self._sync_request('listPublicIpAddresses', params=args)
+        port_forwarding_rules = self._sync_request('listPortForwardingRules')
+
+        try:
+            ip_forwarding_rules = self._sync_request('listIpForwardingRules')
+        except:
```

Comment:
This looks to broad to me. What does it catch? Why would you catch all the things and not fail here?